INTEROPERABLE_DATA.qmd

```diff
@@ -1,67 +1,57 @@
 # Make Your Data Interoperable
 

```

Comment:
I removed the header for 'Taxonomic Authorities', unless we might want to add a description as to what _we_ consider taxonomic authorities in this context?